scripts/install.sh

```diff
@@ -1,17 +1,18 @@
 #!/bin/bash
 
 # Clear screen
+cd ~/
```

Comment:
This makes sure that the script is in the HOME directory before doing anything else.